DeepSpeech.ipynb

```diff
@@ -963,7 +963,7 @@
     "        # Execute operations of tower i on device i\n",
     "        with tf.device(available_devices[i]):\n",
     "            # Create a scope for all operations of tower i\n",
-    "            with tf.name_scope('tower_%d' % i) as scope:\n",
+    "            with tf.variable_scope('tower_%d' % i) as scope:\n",
```

Comment:
@kdavis-mozilla, I've done some more testing and am sure the `tf.name_scope()` was correct to begin with so I've gone ahead and switched this PR back to using it. 

The reason I had originally switched, is that my source install of TensorFlow (technically newer than the 0.12 release) would error out with `tf.name_scope()`. Still have not determined what changed between the 0.12 release and the current state of the source that would cause the usage of `tf.name_scope()` to break like it did.